dockerfiles/Dockerfile.apisix.rpm

```diff
@@ -1,6 +1,6 @@
 ARG image_base="centos"
 ARG image_tag="7"
-ARG checkout_v="2.2"
+ARG checkout_v="2.6"
```

Comment:
We always require to pass a checkout_v. Could we assign it to an empty string?

Reply:
Yes.